src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -70,6 +71,12 @@
     check_type_all_elements_in_iterable,
     min_backend_version,
 )
+from ansys.geometry.core.misc.options import (
```

Comment:
Make sure that the imports are properly done in misc/__init__.py as well ...

However, I'm not sure misc/options is the best place IMO... but I guess it's as good as any location 😄